src/Tizen.NUI.Components/Controls/DropDown.cs

```diff
@@ -384,18 +501,18 @@ public override void ApplyStyle(ViewStyle viewStyle)
             DropDownStyle dropDownStyle = viewStyle as DropDownStyle;
             if (null != dropDownStyle)
             {
-                CreateHeaderText();
-                CreateButtonText();
-                CreateButton();
-
-                CreateListBackgroundImage();
```

Comment:
Please remove `private void CreateListBackgroundImage` function because it is using now.

Reply:
Same as `private void CreateHeaderText()`, `private void CreateButtonText()` and `private void CreateButton()`